src/views/Samples/scss/index.scss

```diff
@@ -0,0 +1,115 @@
+@import "../../../globalStyles/scss/variables";
+
+.sample-page {
+  padding: 17px;
+  background-color: $alabaster;
+}
+
+.collection {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 34px;
```

Comment:
Use the variable $spacers or `rem` units for padding, margin.

Reply:
Done